workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/SpatialOSBuildConfigurationEditor.cs

```diff
@@ -212,237 +573,181 @@ private static TEnum EnumFlagsToggleField<TEnum>(TEnum source)
-
-                    using (IndentLevelScope(indentLevel))
+                    var buildPlatformsString = SelectedPlatformsToString(environmentConfiguration.BuildPlatforms);
+                    var newBuildPlatforms = environmentConfiguration.BuildPlatforms;
+                    var showBuildPlatforms = EditorGUILayout.Foldout(environmentConfiguration.ShowBuildPlatforms,
+                        "Build Platforms: " + buildPlatformsString);
+                    if (showBuildPlatforms)
                     {
-                        controlRect = EditorGUI.IndentedRect(controlRect);
+                        newBuildPlatforms = EnumFlagsToggleField(environmentConfiguration.BuildPlatforms);
                     }
 
-                    if (showIndices)
-                    {
-                        var indexContent = new GUIContent(i.ToString());
-                        var indexRect = new Rect(controlRect) { width = GUI.skin.label.CalcSize(indexContent).x };
-
-                        GUI.Label(indexRect, indexContent);
-
-                        controlRect.x += indexRect.width + 5;
-                        controlRect.width -= indexRect.width + 5;
-                    }
-
-                    var drawerRect = new Rect(controlRect);
-
-                    if (enableReordering)
+                    if (check.changed)
                     {
-                        drawerRect.width -= 40;
-                    }
-
-                    Drawer(drawerRect, item);
-
-                    if (enableReordering)
-                    {
-                        var buttonRect = new Rect(controlRect);
-                        buttonRect.x = buttonRect.xMax - 40;
-                        buttonRect.width = 20;
-
-                        using (new EditorGUI.DisabledScope(i == 0))
-                        {
-                            if (GUI.Button(buttonRect, MoveUpButtonContents))
-                            {
-                                SwapInList(list, i, i - 1);
-                            }
-                        }
-
-                        buttonRect.x += 20;
+                        EditorUtility.SetDirty(target);
+                        Undo.RecordObject(target, "Configure build platforms for worker");
 
-                        using (new EditorGUI.DisabledScope(i == list.Count - 1))
-                        {
-                            if (GUI.Button(buttonRect, MoveDownButtonContents))
-                            {
-                                SwapInList(list, i, i + 1);
-                            }
-                        }
+                        environmentConfiguration.ShowBuildPlatforms = showBuildPlatforms;
+                        environmentConfiguration.BuildPlatforms = newBuildPlatforms;
                     }
                 }
             }
         }
 
-        private static void SwapInList<T>(IList<T> list, int indexA, int indexB)
+        private static IDisposable IndentLevelScope(int increment)
```

Comment:
Is this class still needed?

Reply:
It's still used in one function, I'm going to see if I can redo the layout and get rid of it.